### PR TITLE
feat: random course button

### PR DESCRIPTION
**Pull request by bearsyankees**
raandddoomm

### CODE REVIEW

**Comment by bearsyankees** [comment]
v1.1, not responsive to filters

**Comment by Josh-Cena** [comment]
@bearsyankees Rerun with `./start.sh -f -d` to fetch the latest Ferry schema

**Comment by Josh-Cena** [comment]
Does not look good at a very particular screen width:

image

We should make the button have the same look and feel as the reset button (but different color).

Also I suggest moving it here:

Screenshot 2024-11-23 at 02 01 21

File: frontend/src/components/Search/RandomButton.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.randomButton {
+  font-size: 70%;
+  font-weight: 600;
```
Comment (Josh-Cena) [inline frontend/src/components/Search/RandomButton.module.css:3]:
```suggestion
  font-weight: 500;
```

**Review by Josh-Cena — APPROVED** [review]
:fire:

**Pull request merged**